packages/autocomplete-core/src/__tests__/getInputProps.test.ts

```diff
@@ -1112,23 +1112,405 @@ describe('getInputProps', () => {
       });
 
       describe('Meta+Enter / Ctrl+Enter', () => {
-        test.todo('skips onSelect without item URL');
+        test('skips onSelect without item URL', () => {
+          const onSelect = jest.fn();
+          const navigator = {
```

Comment:
We can extract this to a `createNavigator` test function since we use it everywhere.